h2/utilities.py

```diff
@@ -40,6 +42,11 @@
     b'proxy-authorization', u'proxy-authorization',
 ])
 
+if sys.version_info[0] == 3:
```

Comment:
I marginally prefer to do comparisons against Python 2, and treat that as the special case, because it future-proofs against the major Python version being incremented in future (how does this cope with Python 4?).